### PR TITLE
Person details screen (EXPOSUREAPP-7574, 7657, 7861)

### PR DESCRIPTION
**Pull request by SamuraiKek**
### _Should be merged after https://github.com/corona-warn-app/cwa-app-android/pull/3504_

🗡️  Combined Dagger UI Module for Covid certificates types 
🗑️ `CertificatesFragment` , `VaccinationListFragment`  and its cards , layouts  -> 🆕  `PersonDetailsFragment 🤺 `
🥅  Wire Certificates details Screens 
🖌️  Some refactoring and renaming to fit in `latest` structure
🔧  Bring consistency to main tabs Onboardings in Dark mode , Transitions, Toolbar 

### 🧪 Testing
- Scan Vaccination certificate  1,2 (Blue colour does not mean immune, it means current)  
- Scan Test certificates using Qr-Code
- Register a test and request certificate 

### ⏰ Note
- Recovery certificate screen is wired up ,but repo is not ready yet
- CWA user flag logic is not implemented yet
- Ordering is not implemented yet (Currently first certificate is high prio certificate)
- Proper navigation from `Qr Code scan -> Certificate details -> Person details`  will be addressed in another Ticket 🎟 

### `// ToDo` 
📸  Screenshots in another PR
🧪  Unit tests in another PR 
🦖  Clean up strings in another PR, maybe @axelherbstreith ?

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Comment by SamuraiKek** [comment]
Screen Shot 2021-06-17 at 17 08 52

**Comment by harambasicluka** [comment]
> Scan Vaccination certificate 1,2 (Blue colour does not mean immune, it means current)

Could you please check this with UX, I think in the latest mockup there isn't a "grey state", but IMHO there should be one.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            )
+            else -> context.getString(R.string.vaccination_certificate_incomplete_vaccination)
+        }
+    }
+
+    data class Item(
+        val timeUntilImmunity: Duration?
+    ) : CertificateItem, HasPayloadDiffer {
+        override fun diffPayload(old: Any, new: Any): Any? = if (old::class == new::class) new else null
+        override val stableId = hashCode().toLong()
```
Comment (d4rken) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt:40]: Should the ID change when the time changes?

**Review by d4rken — DISMISSED** [review]
Looks good :+1: 

Further work should be split into a new PR.

**Review by d4rken — APPROVED** [review]
lgtm

**Review by jurajkusnier — APPROVED** [review]
LGTM

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**